stable/prometheus/values.yaml

```diff
@@ -520,6 +520,7 @@ nodeExporter:
 server:
   ## Prometheus server container name
   ##
+  enabled: true
```

Comment:
Please make sure the server default value is true?

Reply:
Oh thats a good catch!
Fixed by changing the option to `disabled: false`, so that the default/empty value will be "enabled"